### ISSUE TITLE
please help check if is all good

### ISSUE THREAD

**Issue by starnazzapa** [pi-node/instructions#115]
![Screenshot 2021-05-05 205525](https://user-images.githubusercontent.com/83729042/117201337-73177d00-ade4-11eb-8299-5fe7644c2e77.png)
![Screenshot 2021-05-05 205429](https://user-images.githubusercontent.com/83729042/117201353-77439a80-ade4-11eb-9881-d9f6b1399dba.png)
can you please tell me if all is good

**Comment by ihatejam** [comment]
Have you done the port check to see if your ports 31400 to 31409 have been successfully forwarded to your node PC?

If so, what were the results?
(You can also use https://www.ipfingerprints.com/portscan.php to double check -- do NOT post any results here without removing/obfuscating your public/external ip address)

**Comment by starnazzapa** [comment]
![image](https://user-images.githubusercontent.com/83729042/117266578-4ea3bb00-ae4d-11eb-9578-e55a813c936e.png)

**Comment by starnazzapa** [comment]
I GIVE PERMISSION ON RUTER AND WINDOWS FIREWALL BUT THIS IS THE RESULT

**Comment by ihatejam** [comment]
If you've set up port forwarding but ipfingerprints still reports that your ports are closed/filtered then there's one more check to do --

Write down the public IP address ipfingerprints thinks you're on [Note: do NOT post your public IP address here].

Compare the public IP address reported by ipfingerprints with the public IP address reported by the WAN interface on your router (you'll need to log into your router config -- and it is NOT the address starting with 192. ). 

If the two IP addresses do not match, then it is likely that your ISP does not hand out "real" public IP addresses. This is common practice in places where the demand for IP addresses is larger than the IP address block allocated to your ISP. To resolve this, you can do the following:
a) ask your ISP if they can give you an IP address that allows port forwarding (warning: their support guys may have no clue what you mean and/or they may not be able to honor your request).
b) switch to an ISP that allows/facilitates port forwarding.
c) use a VPN provider that automatically takes care of port forwarding like [Surfshark](https://surfshark.club/friend/b5BmtEtn?locale=en)

**Comment by starnazzapa** [comment]
![image](https://user-images.githubusercontent.com/83729042/117344264-1a57eb00-ae9d-11eb-8416-8125068c589d.png)
i did the same thing for all ports and this is the result

**Comment by starnazzapa** [comment]
![Screenshot 2021-05-06 210454](https://user-images.githubusercontent.com/83729042/117359200-fac9be00-aeae-11eb-97e1-706bd98adcfc.png)
dont make any sense

**Comment by ihatejam** [comment]
> 
> i did the same thing for all ports and this is the result

If i had to guess, then your consensus container is already running and listening on those 3 ports - so you should be good.

Is that what your troubleshooting screen says? ( i.e. that consensus is running)

**Comment by starnazzapa** [comment]
yes

**Comment by ihatejam** [comment]
You're good to go!!

If it helps, here's an explanation...

You loaded the ipfingerprints site when the consensus container was running (not when the stellar-dummy container aka port-tester was running) so it showed you only 3 ports in use by the consensus container.

When you did the node's built-in port test, the stellar-dummy would have auto-loaded (in place of the consensus container) which should listen on all 10 ports - but the built-in port test is kinda moody and shows some close. Had you visited ipfingerprints at the same time, all 10 ports would have shown as open.

Hope that helps

**Comment by starnazzapa** [comment]
OK WAHT SHOULD HAPPEN NOW MY PI INTAKE SHOULD GROW?

**Comment by ihatejam** [comment]
Nope. Lol. You're running a node for the testnet blockchain - there is zero reward for running it.

**Comment by starnazzapa** [comment]
![image](https://user-images.githubusercontent.com/83729042/117446044-71100400-af33-11eb-8201-6cc05c198de2.png)
nothing will be done

**Comment by ihatejam** [comment]
The node has no effect on your earning.

**Comment by nathusius** [comment]
> 
> 
> OK WAHT SHOULD HAPPEN NOW MY PI INTAKE SHOULD GROW?

LOL!!
